In [4]:
import pandas as pd
import matplotlib as plt
import numpy as np


In [5]:
df = pd.read_excel('/Users/avinashkumar/Downloads/indian_startup_funding_Lab2.xlsx')
df.tail()

Sr No       Date                                       Startup Name  \
106    114 2019-02-08                                            Zilingo   
107    113 2019-02-01                                             FleetX   
108    118 2019-01-04                            Vyome Therapeutics Inc.   
109    119 2019-01-04  Samunnati Financial Intermediation & Services ...   
110    117 2019-01-03                                           CarDekho   

       Industry Vertical                    SubVertical   City  Location  \
106            Ecommerce              Fashion & Apparel  India/Singapore   
107                   AI                      Logistics          Gurgaon   
108  Health and Wellness       Specialty pharmaceutical         India/US   
109              Finance  Non-banking financial company          Chennai   
110           Automobile             Online Marketplace           Jaipur   

                                        Investors Name InvestmentnType  \
106  Sequoia Capital and Temasek Holdings, EDBI, Bu...        Series D   
107  India Quotient and LetsVenture\\xe2\\x80\\x99s...    Pre Series A   
108  Iron Pillar, Perceptive Advisors, Romulus Capi...        Series D   
109                    MASSIF, a Dutch government fund    Debt-Funding   
110  Sequoia India, Hillhouse Capital, Alphabet\\xe...        Series C   

    Amount in USD  Remarks  
106     226000000      NaN  
107   Undisclosed      NaN  
108      22000000      NaN  
109       5000000      NaN  
110     110000000      NaN

In [9]:
df.describe(include="all")

Sr No                           Date Startup Name  \
count   111.000000                            111          111   
unique         NaN                            NaN          106   
top            NaN                            NaN     CarDekho   
freq           NaN                            NaN            3   
mean     63.027027  2019-07-27 09:04:51.891891968          NaN   
min       8.000000            2019-01-03 00:00:00          NaN   
25%      35.500000            2019-05-30 12:00:00          NaN   
50%      63.000000            2019-07-11 00:00:00          NaN   
75%      90.500000            2019-10-03 00:00:00          NaN   
max     119.000000            2020-06-06 00:00:00          NaN   
std      32.233088                            NaN          NaN   

       Industry Vertical SubVertical City  Location           Investors Name  \
count                111         111            111                      109   
unique                53         101             34                      105   
top           E-Commerce   Logistics      Bengaluru  Tiger Global Management   
freq                  11           3             31                        3   
mean                 NaN         NaN            NaN                      NaN   
min                  NaN         NaN            NaN                      NaN   
25%                  NaN         NaN            NaN                      NaN   
50%                  NaN         NaN            NaN                      NaN   
75%                  NaN         NaN            NaN                      NaN   
max                  NaN         NaN            NaN                      NaN   
std                  NaN         NaN            NaN                      NaN   

       InvestmentnType  Amount in USD  Remarks  
count              108          111.0      0.0  
unique              35           79.0      NaN  
top           Series A      5000000.0      NaN  
freq                18            5.0      NaN  
mean               NaN            NaN      NaN  
min                NaN            NaN      NaN  
25%                NaN            NaN      NaN  
50%                NaN            NaN      NaN  
75%                NaN            NaN      NaN  
max                NaN            NaN      NaN  
std                NaN            NaN      NaN


111 rows, 10 columns 
Fix     :   headers
remove :     Sr No   Remarks
check :     InvestmentType - categaorize in series
convert :   Amount in USD (float)
  
check - null values and empty spaces




In [11]:
missing_data = df.isnull()
missing_data.tail(10)

Sr No   Date  Startup Name  Industry Vertical  SubVertical  \
101  False  False         False              False        False   
102  False  False         False              False        False   
103  False  False         False              False        False   
104  False  False         False              False        False   
105  False  False         False              False        False   
106  False  False         False              False        False   
107  False  False         False              False        False   
108  False  False         False              False        False   
109  False  False         False              False        False   
110  False  False         False              False        False   

     City  Location  Investors Name  InvestmentnType  Amount in USD  Remarks  
101           False           False            False          False     True  
102           False           False            False          False     True  
103           False           False            False          False     True  
104           False           False            False          False     True  
105           False           False            False          False     True  
106           False           False            False          False     True  
107           False           False            False          False     True  
108           False           False            False          False     True  
109           False           False            False          False     True  
110           False           False            False          False     True